### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by OlyaPastushchak** [Andriana008/E-learning-Interface-Task1#3]
https://github.com/Andriana008/E-learning-Interface-Task1/blob/47c396ce28611ee2a6353ee6dd917018aa935577/Inheritance_Task1/Inheritance_Task1/Circle.cs#L38-L46

Better to test every function

**Comment by AndriiVaskiv** [comment]
I don't think that it is good idea to test compiler ability to multiply numbers.
